index.html

```diff
@@ -5176,7 +5176,9 @@ <h2 id="AudioWorkletGlobalScope">
           </p>
           <p>
             <a>AudioWorkletGlobalScope</a> has a <dfn>node name to processor
-            definition map</dfn>. Initially it is empty and populated when
+            definition map</dfn>. This map stores definitions of
```

Comment:
Maybe say somewhere here that the map is internal and is not directly exposed to the user?